blog/v4_announcement.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>Could we get rid of this warnings?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc.io/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>